src/rabbit_db_lvc_exchange.erl

```diff
@@ -0,0 +1,129 @@
+%% This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Seriously, a license header from 2012?

Reply:
Whe in doubt, use a license header from any reasonably new file in the core, [such as the `mc` module](https://github.com/rabbitmq/rabbitmq-server/blob/main/deps/rabbit/src/mc.erl#L1-L6).